server/resource_server/config/config.go

```diff
@@ -43,7 +44,7 @@ func SetupDatabaseConnection() *gorm.DB {
 	return db
 }
 
-func CloseDatabaseConnection(db *gorm.DB) {
+func CloseDatabaseConnectionDep(db *gorm.DB) {
```

Comment:
```suggestion
func dep_CloseDatabaseConnection(db *gorm.DB) {
```

Reply:
Done with the suggestions, requesting a review again, actually I also did the go.mod rename from `proxy` to `server` so this is also done 😁